ponycheck/_test.pony

```diff
@@ -1277,3 +1290,132 @@ class iso _AsyncProperty is Property1[String]
 
   fun ref property(arg1: String, ph: PropertyHelper) =>
     _AsyncDelayingActor(ph, _action).do_it()
+
+interface val RndCase[A: Comparable[A] #read]
```

Comment:
The interface and new primitives should all be private so like

_RndCase

I also think it would make sense to call then RandCase or RandomCase.

Reply:
Changed in favor of `_RandomCase`